packages/api-plugin-promotions-coupons/migrations/migrationsNamespace.js

```diff
@@ -0,0 +1 @@
+export const migrationsNamespace = "promotion-coupons";
```

Comment:
So I am not seeing the check for version but we need to do that, but ONLY when they have existing discounts created.